explore-analyze/visualize/charts/metric-charts.md

```diff
@@ -0,0 +1,294 @@
+---
+navigation_title: Metric charts
+applies_to:
+  stack: ga
+  serverless: ga
+description: Instructions and best practices for building metric charts with Kibana Lens in Elastic.
+---
+
+# Build metric charts with {{kib}}
+
+Metric charts make important single values stand out on a dashboard. They're perfect for highlighting KPIs such as error rates or SLOs for example, and for making them understandable at a glance with dynamic coloring or trend indicators.
+
+They work with any numeric data: plain numbers, percentages, or calculations like a count, sum, or average. You can get this numeric data from numeric fields stored in your {{es}} documents, or from aggregation functions and formulas that you can apply to any type of field. <br>
+You can also display strings by using the `Last value` aggregation function that picks up the last document, sorted by timestamp, in the current tine frame.
+
+
+To create metric charts in {{kib}}, you must use **Lens**.
```

Comment:
Could be useful to link to the Lens page here, in case users need a quick definition. 

```suggestion
To create metric charts in {{kib}}, you must use [**Lens**](../lens.md)
```